Xamarin.Forms.Core/Button.cs

```diff
@@ -138,6 +142,23 @@ public ImageSource ImageSource
 			set { SetValue(ImageSourceProperty, value); }
 		}
 
+
+
+		[TypeConverter(typeof(ImageSourceConverter))]
+		public ImageSource LoadingSource
```

Comment:
For now can we not add these to Button.cs? Just use an explicit interface on these for the time being and return null

Reply:
Done